Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterGeoPoint.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Toolkit.Uwp.Services.Twitter
+{
+    class TwitterGeoPoint
```

Comment:
Didn't see this used elsewhere?  Seems like it could be a tuple alias maybe instead of just removed?